duckduckgo/Android
# Add fingerprint protection module

Task/Issue URL: https://app.asana.com/0/488551667048375/1203179499015587/f

### Description
This PR adds:
- Fingerprint protection module
- Feature plugins and content scope plugins for each feature (Battery, canvas, hardware, screen size and temporary storage)

### Steps to test this PR
- Freshly install the app from this branch
- In `fingerprint_protection.db`, verify that the `fingerprinting_battery`, `fingerprinting_canvas`, `fingerprinting_hardware`, `fingerprinting_screen_size` and `fingerprinting_temporary_storage` tables are populated with JSON.
